build-tools/debian-metadata/control

```diff
@@ -0,0 +1,31 @@
+Source: oss-xamarin.android
```

Comment:
No, the Source: line relates to the name of the source package, which is based on the compressed archive name, which is still `oss-xamarin.android_versionnumberstuff`